src/engine/controls/cuecontrol.cpp

```diff
@@ -859,6 +863,10 @@ void CueControl::hotcueSet(HotcueControl* pControl, double value, HotcueSetMode
     mixxx::audio::FramePos cueStartPosition;
     mixxx::audio::FramePos cueEndPosition;
     mixxx::CueType cueType = mixxx::CueType::Invalid;
+    int passStem1Vol;
```

Comment:
What means the pass prefix?

Reply:
pass = pass 2 the 'cuecreation', maybe not clear, sorry